tests/test_parse.py

```diff
@@ -2,40 +2,42 @@
 # -*- encoding: utf-8 -*-
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
 from hl7parser import HL7Message, HL7Segment, HL7Delimiters
 from hl7parser.hl7_data_types import HL7Datetime
 
 import unittest
-
-
-class TestEncoding(unittest.TestCase):
-    """
-    Test correct handling of bytestrings and unicode strings
-    """
-
-    def test_bytestring_segment(self):
-        """ bytestring segment can be cast to unicode and bytestring """
-        segment = HL7Segment(b"FOO|Bääbabamm")
-        str(segment)
-        unicode(segment)
-
-    def test_unicode_segment(self):
-        """ unicode segment can be cast to unicode and bytestring """
-        segment = HL7Segment("FOO|Bääbabamm")
-        str(segment)
-        unicode(segment)
-
-    def test_bytestring_message(self):
-        """ bytestring message can be cast to unicode and bytestring """
-        message = HL7Message(b"MSH|Dingdong the witch is dead\nFOO|Bääbabamm")
-        str(message)
-        unicode(message)
-
-    def test_unicode_message(self):
-        """ unicode message can be cast to unicode and bytestring """
-        message = HL7Message("MSH|Dingdong the witch is dead\nFOO|Bääbabamm")
-        str(message)
-        unicode(message)
+import six
+import sys
+import pytest
+
+@pytest.mark.skipif(sys.version_info.major > 2, reason="not relevant for python 3")
+def test_bytestring_segment():
+    """ bytestring segment can be cast to unicode and bytestring """
+    segment = HL7Segment(b"FOO|Baababamm")
```

Comment:
Why did you remove the Umlauts?

Reply:
It was in an effort to make these tests run in Py3, before I decided they were not relevant. 
Still, having non-ASCII characters in a `b` string doesn't really make sense, does it?

Reply:
Yupp, true for the latter. And they are really not relevant in py3, so we're safe there. In Python2 this should probably be a string-Type and `b'string'` was used to enforce the type in a unicode_literals environment. Test docstring was probably misleading. Here's the original commit: ec632dd1924a5ded35f2155e665d7cc8cd5a27a4